boards/common/esp32s3/include/board_common.h

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2022 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_common_esp32s3
+ * @brief       Common board definitions for ESP32-S3 boards.
```

Comment:
This could be merged with `esp32/include/board_common.h` and `esp32c3/include/board_common.h`.

Reply:
Yes, I know :worried: It is planned for the future to merge all these ESP32x Common Board definitions as much as possible. However, for this PR and the following PR for ESP32-S3, I would prefer to leave it now because it also affects ESP32-C3, which should not be part of this PR.